app/views/header.js

```diff
@@ -0,0 +1,36 @@
+import Mn from 'backbone.marionette';
+import application from '../application';
+import Tracks from '../collections/tracks';
+
+const Header = Mn.ItemView.extend({
+
+    template: require('./templates/header'),
+
+    events: {
+        'click #reset-upnext': 'resetUpNext',
```

Comment:
You should avoid use of `id`s in your markups, except if your absolutely sure that it'll never be duplicated. Prefer the [aria roles](https://www.w3.org/TR/wai-aria/roles) or the `[formaction]` attribute on the `<button>` tags to query your elements.